aries-site/src/pages/templates/content-layouts.mdx

```diff
@@ -32,39 +71,89 @@ Specifying layout content in this way will also ensure that it is interepreted a
+Key aspects to the approach are:
+- At larger breakpoints, the content layout will be a two-column layout; a single column 
+at smaller breakpoints.
+- The filters region will be fixed-width when in a two-column layout and [flex-width](/components/grid#flexible-columns) 
+when in a single column. The results region always uses a [fluid grid](/components/grid#fluid-grids).
+- The filters region are prioritized over the results region because the filters are needed to control 
+the results.
+
+<Example
+  showResponsiveControls={['fullScreen']}
+  code={[
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/ContainerDrivenLayout.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/PageHeader/PageHeader.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/AppResults.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/FilterPanel.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/data/mockData/applications.json",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/constants.js",
+    ]}
+  github="https://github.com/grommet/hpe-design-system/blob/master/aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout.js"
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  height="650px"
+  width="100%"
+  screenContainer={{ scale: 0.75 }}
+>
+  <ContainerDrivenLayout />
+</Example>
 
-See the table below for breakpoints defined by the HPE Design System. Keep in mind that these breakpoints are not meant to target every single existing device &mdash; we understand that technology is ever evolving. Instead, the ranges defined here help build a strong foundation to account for diverse use cases when creating responsive designs.
+### Content-Driven Layouts
+Content-driven layouts are driven from the inside-out, allowing the container to respond 
+and adapt to things like user entered content, like names. The layout is custom fit to 
+the content it holds.
 
-| T-shirt Size   | Range         |  Margin (pixels) |
-| -------------- | ----------    | -----------------:
-| **xsmall**    | 576 and below |      12
-| **small**     | 577 - 768     |      24
-| **medium**      | 769 - 1080    |      24
-| **large**     | 1081 - 1439   |      48
-| **xlarge**      | 1440 and above|      48
+The content-driven approach is useful for layouts such as modal dialogs or side-drawer 
+content within a layer, where the content should occupy the space it needs and not be 
+artificially constrained by its layout container.
```

Comment:
Suggestion: "artificially" -> "awkwardly", or something that communicates more the potential result. As an example, if you saw a Layer where the Heading text wrapped for a few characters, would you say it looked "artificial" or would you say it looked "awkward" or "appeared broken"?

Reply:
done